### PR TITLE
Update team page and move to latest Middleman/Ruby

### PR DESCRIPTION
**Pull request by ndorfin**
pug_dance

### CODE REVIEW

File: spec/features/error_handler_spec.rb [hunk 1]
```diff
@@ -1,14 +0,0 @@
-require "spec_helper"
-
-describe "error handler", js: true, type: :feature, js_errors: true do
```
Comment (nigelramsay) [inline spec/features/error_handler_spec.rb]: Why are these specs being deleted?

**Comment by nigelramsay** [comment]
Looks good, other than the removal of the specs. Can they be restored?

File: spec/features/error_handler_spec.rb (hunk 1 above)
Comment (ndorfin) [inline spec/features/error_handler_spec.rb]: They're not adding any value.
They're cumbersome.
They don't work with JavaScript, and need to be replaced with something that actually works. I had a ticket on the trello board asking for help with finding a working alternative, but that was archived for some reason.

**Comment by nigelramsay** [comment]
Ship it @ndorfin

**Pull request merged**